amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -2861,6 +2856,10 @@ type ColonyAction @model @searchable {
   Metadata associated with the action (Eg. Custom action title)
   """
   metadata: ColonyActionMetadata @hasOne(fields: ["id"])
+  """
+  The reputation root hash at the time of the creation of the action
+  """
+  rootHash: String!
```

Comment:
Uff. I just realized. This needs a migration script, otherwise, once deployed to production, it will instantly break all existing motions.

Reply:
@rdig @ArmandoGraterol I think this needs to be done as a two-step migration to avoid downtime, the first step would be to add this field as non-nullable, migrate the existing data, then make it non-nullable in the schema.